Resources/Public/JavaScript/Main.js

```diff
@@ -0,0 +1,36 @@
+document.addEventListener("DOMContentLoaded", function () {
+    var t = document.querySelectorAll(".tx-tea table"),
+        i = t.length;
+    while (--i >= 0) makeSortable(t[i]);
```

Comment:
Please let's have the `{` and `}` here. (Shouldn't eslint have complained about this?)